### PR TITLE
update doc in response to changes to install

### PR DESCRIPTION
**Pull request by kalantar**
Addresses issue https://github.com/iter8-tools/iter8/issues/615

### CODE REVIEW

File: mkdocs/docs/tutorials/knative/canary-fixedsplit.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ You will create the following resources in this tutorial.
     - eventually replaces `baseline` with `candidate` using Kustomize
 
 ???+ warning "Before you begin, you will need... "
-    **Kubernetes cluster:** Ensure that you have Kubernetes cluster with Iter8 and Knative installed. You can do this by following Steps 1, 2, and 3 of the [quick start tutorial for Knative](../../../getting-started/quick-start/with-knative/).
+    **Kubernetes cluster:** Ensure that you have a Kubernetes cluster with Iter8, Knative and the sample metrics for Knative (which assume Prometheus) installed. You can do this by following Steps 1, 2, and 3 of the [quick start tutorial for Knative](../../../getting-started/quick-start/with-knative/).
```
Comment (sriumcp) [inline mkdocs/docs/tutorials/knative/canary-fixedsplit.md]: Ensure that you have a Kubernetes cluster with Iter8, Knative, Iter8 sample metrics for Knative, and the Iter8 Prometheus add-on  installed.

File: mkdocs/docs/tutorials/knative/canary-progressive.md [hunk 2]
```diff
@@ -19,15 +19,23 @@ You will create the following resources in this tutorial.
...
     **Cleanup:** If you ran an Iter8 tutorial earlier, run the associated cleanup step.
 
     **ITER8:** Ensure that `ITER8` environment variable is set to the root directory of your cloned Iter8 repo. See [Step 2 of the quick start tutorial for Knative](../../../getting-started/quick-start/with-knative/#2-clone-iter8-repo) for example.
 
     **[Helm v3](https://helm.sh/) and [`iter8ctl`](../../../getting-started/install/#optional-step-3-iter8ctl):** This tutorial uses Helm v3 and `iter8ctl`.
 
-## 1. Create versions
+## 1. Give Permission to Iter8 to Call `helm upgrade`
+
+The final step of the experiment is to promote the winning version. In this sample experiment, this is done using `helm upgrade`. Helm uses secrets to record information about an installation. Iter8 must have permission to these resources.
```
Comment (sriumcp) [inline mkdocs/docs/tutorials/knative/canary-progressive.md]: Helm uses secrets to record information about releases. This tutorial uses an experiment that invokes Helm. Enable this experiment using the following RBAC.

File: samples/knative/quickstart/platformsetup.sh [hunk 3]
```diff
@@ -30,6 +30,7 @@ if [[ ! " ${NETWORK_LAYERS[@]} " =~ " ${1} " ]]; then
 fi
 
 # Step 1: Export correct tags for install artifacts
+export GIT_ACCOUNT="${GIT_ACCOUNT:iter8-tools}"
```
Comment (sriumcp) [inline samples/knative/quickstart/platformsetup.sh]: Why is there a minus in lines 34 and 35 but none in 33?

**Review by sriumcp — CHANGES_REQUESTED** [review]
Are there additional changes needed for automated and manual test files?

**Comment by kalantar** [comment]
@sriumcp thanks for initial comments; now addressed. A question I have is whether the way metrics are installed -- in platformsetup.sh (https://github.com/kalantar/iter8/blob/install-changes/samples/knative/quickstart/platformsetup.sh#L123) is appropriate vs an explicit step in the quickstart. I see little value in an explicit step.

File: samples/knative/quickstart/platformsetup.sh (hunk 3 above)
Comment (kalantar) [reply samples/knative/quickstart/platformsetup.sh]: my mistake. There should be a "-"; see https://stackoverflow.com/a/48829326/3482067

**Comment by sriumcp** [comment]
**Some thoughts on metrics...**

A/B and A/B/n experiments are all about business metrics, or in ML serving use-cases, ML specific metrics. In A/B, A/B/n experiments with business/ML metrics, we will want to show how custom (business/ML) Iter8 metrics are created, installed and used as part of the experiment.

For Canary/Conformance experiments, perhaps we can simply say, when you did the "easy install" of Iter8 as part of quick start, sample metrics were installed along with Prometheus add-on;  we will use them in this experiment.

Our A/B, and A/B/n  experiments so far have involved a combination of business  + system metrics (latency/error-rate). So, even in A/B and A/B/n, some metrics are available as part of easy install, and others need to be shown as part of the tutorial.

No metrics == no experiments... so, the metrics story is just as integral to each tutorial as the experiment story (although, so far we haven't made an effort to say that story).

**Comment by kalantar** [comment]
> No metrics == no experiments... so, the metrics story is just as integral to each tutorial as the experiment story (although, so far we haven't made an effort to say that story).

For Istio, I tested Istio A/B/n example with productpage with a business reward. We can include this metric in the tutorial as an initial place to tell this story.

**Comment by kalantar** [comment]
Not sure why the merge introduced so many file changes. Closing and recreating to focus on the relevant changes.

**Pull request closed without merging**